spec/models/user_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user) { User.create(name: 'Joy', photo: 'https://www.google.com/url', bio: 'she is a software developer', posts_counter: 0) }
+
+  it 'Name should not be blank' do
+    user.name = nil
+    expect(user).to_not be_blank
+  end
```

Comment:
- [ ] Kindly add tests for all validations in the `User` model as requested by the previous reviewer and as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.). Currently there are no tests for `PostsCounter must be an integer greater than or equal to zero.` validation.